handler/create_user/create_user.go

```diff
@@ -0,0 +1,89 @@
+package handler
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+type createUserRequest struct {
+	UserId string `json:"user_id"`
+}
+
+func CreateUser(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	var reqBody createUserRequest
+
+	// TODO: Put error handling in helper function
+	dec := json.NewDecoder(r.Body)
```

Comment:
do you need to defer closing the request body?

Reply:
![image](https://github.com/sunb26/xat/assets/56877739/cf5bf0cb-c73a-4153-868a-51d727b53a0a)
I think it should do it automatically? Unless I'm interpretting this wrong lol which I could be.

Reply:
you're right, thanks for confirming